Bug 1364170 - Fill out event dataset documentation (#536)
mozilla/firefox-data-docs: src/datasets/batch_view/events/reference.md

# Events

> This derived dataset makes it easier to analyze the [event ping][event_ping].

Author: @badboy's comment made me realize we should call this out explicitly.

```suggestion
This derived dataset makes it easier to analyze the Firefox [event ping][event_ping].
```

It has the following advantages over accessing the raw ping table (`telemetry.event`):

- There is no need to `UNNEST` the `events` column: this is already done for you.

> - You don't have to introspect a BigQuery structure to find out which process emitted an event: you can query the `event_process` column directly.

Reviewer: This might state the advantage more clearly:

```suggestion
- You don't have to know which process type emitted your event. If you care, you can query the `event_process` column.
```
